leakybucket_test.go

```diff
@@ -48,7 +49,7 @@ func (s *LimitersTestSuite) TestLeakyBucketRealClock() {
 	rate := time.Millisecond * 10
 	clock := l.NewSystemClock()
 	for _, requestRate := range []time.Duration{rate / 2} {
-		for name, bucket := range s.leakyBuckets(capacity, rate, clock) {
+		for name, bucket := range s.leakyBuckets(capacity, rate, time.Minute, clock) {
```

Comment:
Here and in other similar places below (and other test files): if a TTL is irrelevant for this test does it make sense to set it to `0`?